demo/src/plugins/dashboard/components/active-edits/index.jsx

```diff
@@ -0,0 +1,114 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+import { connect } from 'react-redux';
+import Message from '@gdbots/pbj/Message';
+import { STATUS_PENDING } from '@triniti/app/constants';
+import {
+  Button,
+  Card,
+  CardBody,
+  CardHeader,
+  Icon,
+  StatusMessage,
+  Table,
+} from '@triniti/admin-ui-plugin/components';
+import delegate from './delegate';
+import selector from './selector';
+import TableRow from './TableRow';
+
+class ActiveEdits extends React.Component {
+    static propTypes = {
+      accessToken: PropTypes.string.isRequired,
+      collaborationNodes: PropTypes.arrayOf(PropTypes.instanceOf(Message)).isRequired,
+      contentType: PropTypes.string,
+      handleRequestCollaborationNodes: PropTypes.func.isRequired,
+      title: PropTypes.string.isRequired,
+    };
+
+    static defaultProps = {
+      contentType: null,
+    };
+
+    constructor(props) {
+      super(props);
+      this.handleRequestCollaborationNodes = this.handleRequestCollaborationNodes.bind(this);
+      this.state = { status: null };
+    }
+
+    componentDidMount() {
+      this.handleRequestCollaborationNodes();
+    }
+
+    async handleRequestCollaborationNodes() {
+      const { handleRequestCollaborationNodes, accessToken } = this.props;
+      this.setState({ status: STATUS_PENDING });
+      await handleRequestCollaborationNodes(accessToken);
+      this.setState({ status: null });
```

Comment:
@nanetteCodes oh you can use STATUS_FULFILLED here, just dont set it as the initial state (since its not fulfilled on mount)